src/client/calendar/events/event-details/EventDetails.jsx

```diff
@@ -337,13 +341,14 @@ export default connect(
     // calendar: getCalendar()
 
     const event = getEvent(state, ownProps.params.eventId)
+    const calendar = getCalendar(state, {calendarId: event.calendarId})
```

Comment:
I think we should be consistent with simple selectors like that and either use named arguments or don't. Above selector doesn't use those since it's pretty self-explanatory when you see like like this:

`const calendar = getCalendar(state, event.calendarId)`

Thoughts?

Reply:
Yes, but `getCalendar` selector was alredy their like that, so I reused without changing it's params. I can refactor this, but maybe it will be better to do it in separate PR?

Reply:
@graymur yeah, I know, it just became apparent here. I am fine with doing it in another PR.